packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -793,7 +793,7 @@ export interface CallAdapterCallManagement extends CallAdapterCallOperations {
    *
    * @public
    */
-  joinCall(microphoneOn?: boolean): Call | undefined;
+  joinCall(microphoneOn?: boolean, cameraOn?: boolean): Call | undefined;
```

Comment:
A list of optional arguments is bad, but we can't change the `microphoneOn` one now..

I approve this change. It might be worth asking sdk reviewers on the side if its better to have what we have, or should we introduce another function instead like:
```ts
joinCallWithOptions(options: {microphoneOn?: boolean, cameraOn?: boolean});
```

and have:

```ts
joinCall(); // new function
joinCall(microphoneOn?: boolean) // mark as deprecated
joinCallWithOptions(options) //new function
```

But don't block PR on this

Reply:
I do like the idea of overloads here as well, the options bag version would be a good why to bring us closer to the SDK without all the complexity of needing the streams to do it.